prepare-docs.rb

```diff
@@ -5,38 +5,100 @@
 require "net/http"
 require "uri"
 require "rexml/document"
-# require "nokogiri"
+require 'json'
+require 'open-uri'
 
 $docs_repo = "https://github.com/openhab/openhab-docs"
 $docs_repo_root = $docs_repo + "/blob/main"
 $docs_repo_branch = "final"
 $addons_repo_branch = "main"
 $version = nil
+$verbose = false
 
 $ignore_addons = ['transport.modbus', 'transport.feed', 'javasound', 'webaudio', 'oh2']
 
+def verbose(message)
+    if ($verbose == true) then
+      puts message
+    end
+end
+
+def checkout_pull_request(pr, target_directory)
```

Comment:
Probably a bug 🐞, or at least unnecessary - I also do not see the need for this ... I'll remove and check if it will still work (it actually should) ...